daemon/models/custom.py

```diff
@@ -72,7 +72,7 @@ def get_pydantic_fields(config: Union[dict, argparse.ArgumentParser]):
             if hasattr(arg_type, '__self__'):
                 arg_type = type(arg.default) if arg.default else int
             arg_type = str if isinstance(arg_type, argparse.FileType) else arg_type
-            arg_type = dict if type(arg) == KVAppendAction else arg_type
+            arg_type = dict if (isinstance(arg, KVAppendAction) or isinstance(arg, DockerKwargsAppendAction)) else arg_type
```

Comment:
You can also do `isinstance(arg, (KVAppendAction, DockerKwargsAppendAction))` :)